onnxruntime/test/framework/nuphar/nuphar_fuse_op_test.cc

```diff
@@ -0,0 +1,343 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/common/status.h"
+#include "core/common/logging/logging.h"
+#include "core/framework/execution_provider.h"
+#include "core/framework/op_kernel.h"
+#include "core/framework/session_state.h"
+#include "core/framework/tensorprotoutils.h"
+#include "core/graph/graph.h"
+#include "core/graph/model.h"
+#include "core/graph/op.h"
+#include "core/providers/cpu/math/element_wise_ops.h"
+#include "core/session/inference_session.h"
+#include "gtest/gtest.h"
+#include "test/capturing_sink.h"
+#include "test/framework/test_utils.h"
+#include "test/test_environment.h"
+#include "test/util/include/default_providers.h"
+
+using namespace std;
+using namespace ONNX_NAMESPACE;
+
+namespace onnxruntime {
+namespace test {
+// Nuphar Fused Op Test
+TEST(NupharTest, FuseAddTest) {
+#ifdef USE_NUPHAR
+  // Create Model
```

Comment:
Why have #ifdef's inside each test vs outside of all the tests?

Reply:
Seems stale code, I'll remove them.

---
In reply to: [312423814](https://github.com/microsoft/onnxruntime/pull/881#discussion_r312423814) [](ancestors = 312423814)